ExcelReader/Controllers/ITRequestController.cs

```diff
@@ -10,24 +10,24 @@ namespace ExcelReader.Controllers
     public class ITRequestController : Controller
```

Comment:
Naming convention is wrong. It should be "ItRequestWithFileController"

Reply:
done